_pages/home/updating-to-version-2211.35/modernize-apps-migrated-from-6.8-to-2211.19.md

```diff
@@ -1,466 +1,473 @@
+   import { existsSync } from 'fs';
+   ```
 
-1. Change the the export path of the `AppServerModule` from `./app/app.server.module'` to `./app/app.module.server'`. 
-2. And export `AppServerModule` using `as default`.
+1. In `server.ts`, add the following imports to the top of the file:
 
-```diff
-- export { AppServerModule } from './app/app.server.module';
-+ export { AppServerModule as default } from './app/app.module.server';
-```
+   ```ts
+   import { APP_BASE_HREF } from '@angular/common';
+   import {
+     NgExpressEngineDecorator,
+     ngExpressEngine as engine,
+   } from '@spartacus/setup/ssr';
+   import express from 'express';
+   import { dirname, join, resolve } from 'node:path';
+   import { fileURLToPath } from 'node:url';
+   import AppServerModule from './src/main.server';
+   ```
 
-### `server.ts`
+1. In `server.ts`, remove the following constants:
 
-1. Change the imports in the top of the file according to the following diff:
+   ```ts
+   const distFolder = join(process.cwd(), 'dist/YOUR-APP-NAME/browser');
+   const indexHtml = existsSync(join(distFolder, 'index.original.html'))
+     ? 'index.original.html'
+     : 'index';
+   ```
 
-```diff
-- import 'zone.js/node';
+1. In `server.ts`, add the following constants:
 
-- import { ngExpressEngine as engine } from - '@spartacus/setup/ssr';
-- import { NgExpressEngineDecorator } from - '@spartacus/setup/ssr';
-- import * as express from 'express';
-- import { join } from 'path';
+   ```ts
+   const serverDistFolder = dirname(fileURLToPath(import.meta.url));
+   const browserDistFolder = resolve(serverDistFolder, '../browser');
+   const indexHtml = join(browserDistFolder, 'index.html');
+   ```
 
-- import { AppServerModule } from './src/main.- server';
-- import { APP_BASE_HREF } from '@angular/common';
-- import { existsSync } from 'fs';
+1. In the `server.set('views'` call of `server.ts`, use the `browserDistFolder` constant as a second argument, instead of `distFolder`.
 
-+ import { APP_BASE_HREF } from '@angular/common';
-+ import {
-+   NgExpressEngineDecorator,
-+   ngExpressEngine as engine,
-+ } from '@spartacus/setup/ssr';
-+ import express from 'express';
-+ import { dirname, join, resolve } from 'node:path';
-+ import { fileURLToPath } from 'node:url';
-+ import AppServerModule from './src/main.server';
-```
+   The updated call should appear as follows:
 
-2. Replace two old constants: `distFolder` and `indexHtml` with new three constants: `serverDistFolder`, `browserDistFolder`, `indexHtml`, according to the diff below:
+   ```ts
+   server.set('views', browserDistFolder);
+   ```
 
-```diff
-- const distFolder = join(process.cwd(), 'dist/YOUR-APP-NAME/browser');
-- const indexHtml = existsSync(join(distFolder, 'index.original.html'))
--   ? 'index.original.html'
--   : 'index';
+1. In the `express.static(` call of `server.ts`, please use the `browserDistFolder` constant as an argument, instead of `distFolder`.
```

Comment:
```suggestion
1. In the `express.static(` call of `server.ts`, please use the `browserDistFolder` constant as the first argument, instead of `distFolder`.
```